packages/medusa/src/api/routes/store/payment-collections/get-payment-collection.ts

```diff
@@ -0,0 +1,20 @@
+import { PaymentCollectionService } from "../../../../services"
+import { FindParams } from "../../../../types/common"
+
+export default async (req, res) => {
+  const { id } = req.params
+  const retrieveConfig = req.retrieveConfig
+
+  const paymentCollectionService: PaymentCollectionService = req.scope.resolve(
+    "paymentCollectionService"
+  )
+
+  const payment_collection = await paymentCollectionService.retrieve(
```

Comment:
nit(non-blocking): Camel case to follow general convention
```suggestion
  const paymentCollection = await paymentCollectionService.retrieve(
```